### PR TITLE
Mapping to NEB supercell calcuations

### PR DESCRIPTION
**Pull request by jmmshn**
Changes:
- Added functions for creating supercells for neb migrations analysis.  Tries to construct the most cubic supercell possible given some constraints.
- That supercell mapping can be used to create the initial and final simulation cells for neb or approxNEB calculations
- Added a `get_sc_structures` function to `MigrationHop` that creates a NEB simulation supercell using the data on a `MigrationHop`

### CODE REVIEW

**Comment by jmmshn** [comment]
@shyuep  this is nearly complete but it requires changes from PR #173 to compare against.

**Comment by shyuep** [comment]
Ok, I just merged 713.

**Comment by jmmshn** [comment]
thanks!  I will clean this up a bit and remove the WIP.

File: pymatgen_diffusion/neb/pathfinder.py [hunk 1]
```diff
@@ -314,7 +323,7 @@ def __init__(
         isite: Site,
         esite: Site,
         symm_structure: SymmetrizedStructure,
-        symprec: float = 0.01,
+        symprec: float = 0.001,
```
Comment (jmmshn) [inline pymatgen_diffusion/neb/pathfinder.py:326]: matches the default value from pymatgen

**Comment by jmmshn** [comment]
@shyuep is it possible to ignore the maggma.py file?
It does contain any logic needed for any of the migration graph analysis.
It just has some simple functions to help with database queries.

**Pull request merged**